ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1202,3 +1202,24 @@ literals:
 - thermal_energy_accumulator
 - ground_isolation_command
 - ground_temperature_sensor
+
+## New Fields for PAC (ACU) Units
+- return_air_isolation_damper_status
+	- OPEN
+	- CLOSE
+- alert_reset_command
```

Comment:
I am not convinced this is necessary. See issue 502.

Reply:
alert_reset_command is deleted and updated in the pull request